server/budgettracker.common/Models/User.cs

```diff
@@ -8,10 +9,12 @@ namespace budgettracker.common.Models
     /// </summary>
     public class User : IUser
     {
+        public int Id { get; set; }
```

Comment:
This should be a Guid

Reply:
I agree, but this is not the schema right now. We will use guids and I have an [issue for that here](https://github.com/ianbro/BudgetTracker/pull/38).

However, that would require a refactor in this issue, which is outside of the scope of this MR. So the next issue I'm going to do is issue #38 but right now, I'm still using Ids because that's what the model is already using and I want to stay within the scope of this issue.